### ISSUE TITLE
Mahapps Metro themes overwrite Label foreground colors even when explicitly set

### ISSUE THREAD

**Issue by bloritsch** [MahApps/MahApps.Metro#2201]
I had some labels that were not respecting the colors I explicitly set with code that looked like this:

```XAML
<Label Foreground="White" Content="Fu"/>
<Label Foreground="Red" Content="Bar"/>
<Label Foreground="Green" Content="Snap"/>
```
After applying the themes, those labels were set to all Black when the BaseLight theme was applied and all white when the BaseDark theme was applied.

I would expect them to change colors if I left the Foreground property alone, but not when I explicitly set the colors.  I had to change them all to `TextBlock` objects to get them to render correctly, but that also forced me to do some other layout changes since the code took advantage of the `HorizontalContentAlignment` and `VerticalContentAlignment` to position those labels free form as the user dragged the mouse.

**Comment by punker76** [comment]
@bloritsch I tested your 3 Labels in the main demo and it works...

![2015-11-05_21h32_54](https://cloud.githubusercontent.com/assets/658431/10980926/e274ac50-8404-11e5-9387-dd24db90ef4f.png)

Which version of MahApps do you use?
Is there any special style in your app that possible override some stuff?

**Comment by bloritsch** [comment]
According to the details property it's Mahapps.Metro 1.1.2

There's no style that blanket applies to all Labels (there's a lot of XAML
so I may have missed something in my last comb through to enable "night
mode").

**Comment by punker76** [comment]
@bloritsch ok, so please try the latest alpha. thx

**Comment by bloritsch** [comment]
Do I just grab the source and build it to get the latest alpha?

**Comment by punker76** [comment]
@bloritsch no, the alpha is also available via NuGet

`Install-Package MahApps.Metro -Pre`

or via NuGet ui...

**Comment by punker76** [comment]
@bloritsch have you tried the latest alpha?

**Comment by bloritsch** [comment]
I haven't had the opportunity yet.

**Comment by bloritsch** [comment]
I finally tracked down the problem, and it was in our app.  So one of the most surprising things for me was that we had a code block like this before we were able to go hog wild with Mahapps.Metro:

    <Style TargetType="{x:Type TextBlock}">
        <Setter Property="FontWeight" Value="Bold"/>
        <Setter Property="FontFamily" Value="{StaticResource ContentFontFamily}"/>
        <Setter Property="FontSize" Value="{StaticResource ContentFontSize}"/>
        <Setter Property="SnapsToDevicePixels" Value="2"/>
        <Setter Property="Background" Value="Transparent"/>
        <Setter Property="Foreground" Value="{DynamicResource BlackBrush}"/>
    </Style>

The funny thing is that this forced all <Label>s to go with "BlackBrush" (depending on the dark/light theme it would always be white/black).  Makes me wonder if .Net uses a <TextBlock> inside a <Label>....

XAML Spy was useless for finding this little tidbit out, as I started a new project and just started adding bits and snatches of the styles until I reproduced the issue.  NOTE: this same issue also affects the text in <MenuItem> classes.  That means no matter what Foreground property set by the MenuItem theme (i.e. disabled menu items) they were always rendered as if they were enabled.

Sorry for wasting your time, but hopefully this bit of trivia will help someone besides myself.
